src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+// ACTIONS
+const STATUS = 'my-app/widgets/STATUS';
+
+export default function categoriesReducer(state = {}, action = {}) {
+  switch (action.type) {
+    case STATUS:
+      return [
+        {
+          status: 'Under construction',
+        },
+      ];
+
+    default:
+      return state;
+  }
+}
+
+export const statusWidgets = (widget) => ({ type: STATUS, widget });
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you rename this function to an appropriate name.